hsreplaynet/static/scripts/src/components/discover/ClusterSignature.tsx

```diff
@@ -4,26 +4,76 @@ import { SortDirection } from "../../interfaces";
 import CardData from "../../CardData";
 import { ArchetypeSignature } from "../../utils/api";
 import LoadingSpinner from "../LoadingSpinner";
+import { CardTableColumnKey } from "../tables/cardtable/CardTableColumns";
+import Checkbox from "../Checkbox";
+import { fetchCSRF } from "../../helpers";
 
 interface Props {
 	cardData: CardData | null;
+	clusterId: string;
+	format: string;
+	playerClass: string;
+	requestReload: () => void;
+	requiredCards: Array<Number>;
```

Comment:
`Array<x>` is equivalent to `x[]` - we tend to use the latter in hsreplaynet.
In addition, Number is once again not the same as number :-)

```suggestion
	requiredCards: number[];
```